client/ng/components/accounts/login/login.html

```diff
@@ -0,0 +1,44 @@
+<!-- Login -->
```

Comment:
Znowu wcięcia. Jak zmienisz na 2 spacje to zobaczysz jaka jest różnica w czytelności.